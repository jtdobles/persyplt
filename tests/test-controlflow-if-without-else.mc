void main(){
	int a = 8;

	if (a > 1) {
		prints('a is greater than 1');
	}
}