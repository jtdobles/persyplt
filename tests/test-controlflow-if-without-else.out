'a is greater than 1'
